src/sequence_batch.h

```diff
@@ -81,6 +81,25 @@ class SequenceBatch {
     return negative_sequence_batch_[sequence_index];
   }
 
+  // big_endian: N_pos is in the order of sequence
+  // little_endian: N_pos is in the order from the sequence right side to left, 
+  //                this is the order of the GenerateSeed
+  inline void GetSequenceNsAt(uint32_t sequence_index, bool little_endian, std::vector<int> &N_pos) {
+    int l = sequence_batch_[sequence_index]->seq.l;
+    char *s = sequence_batch_[sequence_index]->seq.s;
+    int i;
```

Comment:
Better to avoid this kind of variables without initial value. It can be dangerous as we saw previously. For for loop, just declare the variable in the for statement, if it is not used after the loop.

Reply:
Done.